modules/ocf_syslog/files/ocf.conf

```diff
@@ -15,41 +25,46 @@ template(name="ocf-remote" type="list") {
     constant(value="\n")
 }
 
-# template for errored remote log lines
-template(name="ocf-errors" type="list") {
-    constant(value="ERRORED_TAG[")
-    property(name="syslogtag")
-    constant(value="]  MESSAGE[")
-    property(name="timegenerated" dateFormat="rfc3339")
-    constant(value=" ")
-    property(name="hostname")
-    constant(value=":  ")
-    property(name="msg")
-    constant(value="]\n")
+# template for remote docker log filenames
+template(name="ocf-docker-file" type="list") {
+    constant(value="/var/log/remote/services/")
+    # in jessie, syslogtag has a weird ":" appended, so we have to regex that out;
+    # the regex bits can be removed when we're totally on stretch.
+    property(name="syslogtag" regex.Expression="^[^:]+" regex.Type="ERE" regex.Match="0" securepath="replace")
+    constant(value="/")
+    property(name="syslogtag" regex.Expression="^[^:]+" regex.Type="ERE" regex.Match="0" securepath="replace")
+    constant(value=".log")
 }
 
 # template for remote log filenames
 template(name="ocf-remote-file" type="list") {
     constant(value="/var/log/remote/")
-    # in jessie, syslogtag has a weird ":" appended, so we have to regex that out;
-    # the regex bits can be removed when we're totally on stretch.
-    property(name="syslogtag" regex.Expression="^[^:]+" regex.Type="ERE" regex.Match="0" SecurePath="replace")
+    property(name="hostname" securepath="replace")
+    constant(value="/")
+    property(name="programname" caseconversion="lower" securepath="replace")
+    constant(value="/")
+    property(name="programname" caseconversion="lower" securepath="replace")
```

Comment:
Same here with the duplication of the program name.

Reply:
Generating a combined syslog would be an exercise left to the reader...

Reply:
You can do something like `tail -f */*.log` to show a stream of the incoming logs for a host, but then it also shows each of the names of the logs it is printing from. I think from an organization perspective though this is better than having one really long log that's harder to find things in or one directory with a ton of logs ending in `.13.gz` or something. Even a `grep -r` is pretty easy to do, but if others prefer a flatter organization of logs then I'd be ok with that.